packages/parser/package.json

```diff
@@ -28,7 +28,7 @@
     "url": "https://github.com/typescript-eslint/typescript-eslint/issues"
   },
   "homepage": "https://typescript-eslint.io/packages/parser",
-  "license": "BSD-2-Clause",
+  "license": "MITClause",
```

Comment:
@JoshuaKGoldberg @bradzacher this looks like a typo:

```diff
-  "license": "MITClause",
+  "license": "MIT",
```

Reply:
PR to fix this: https://github.com/typescript-eslint/typescript-eslint/pull/10481